crest/Assets/Crest/Crest-Examples/Shared/Scripts/CamController.cs

```diff
@@ -45,6 +45,9 @@ public class CamController : MonoBehaviour
     [System.Serializable]
     class DebugFields
     {
+        [Tooltip("Disables controller preventing the camera from rolling (rotating on the z axis).")]
+        public bool disableCameraRollPrevention = false;
```

Comment:
"disable prevention" -> tempted to invert: "enable camera roll". but debug code so doesnt matter too much!

Reply:
I agree that would be much nicer wording. For debug options I believe they should be checked to be enabled (ie always disabled by default) from a UX perspective.

Reply:
I'm not a fan of "prevention". Does disableCameraRollClamp or disableCameraRollConstraint sound better?

Reply:
i feel like "enable camera roll" is disabled by default, and checked to be enabled. but disableCameraRollClamp is probably my favorite of the lot so thumbs up for that :)

Reply:
Oh... I feel dumb :^). Sorry. I have renamed to "enable camera roll"